actors/builtin/multisig/multisig_actor.go

```diff
@@ -64,9 +64,9 @@ func (a Actor) Exports() []interface{} {
 var _ abi.Invokee = Actor{}
 
 type ConstructorParams struct {
-	Signers               []addr.Address
+	Signers               []addr.Address // PARCHECK unbounded
 	NumApprovalsThreshold uint64
-	UnlockDuration        abi.ChainEpoch
+	UnlockDuration        abi.ChainEpoch // PARCHECK unbounded
```

Comment:
I assume these are fine, but I wanted to check to see if there are any constraints in the crypto on the max number of signers.

Reply:
I noted this in #897